### DISCUSSION TITLE
Update the sample applications to the SDK project format?

### DISCUSSION THREAD

**Discussion by Numpsy** [icsharpcode/SharpZipLib#526] [Ideas]
A thought from #525 - I wondered previously if the sample projects could/should be updated to PackageReference format (rather than packages.config) and/or converted to SDK format, but wasn't sure because that would reduce support for older versions of Visual Studio.

However, you could say that doesn't matter so much and using the new format is better going forwards because it should have better support for running the samples on non-Windows systems?

**Comment by piksel** [comment]
Yeah, and they are also much easier to maintain. The old format would still be available in older tags for older environments.
